package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pagopa/io-spid-commons",
-  "version": "6.5.0",
```

Comment:
We don't have to change version here, we bump it at deploy time

Reply:
you are right, thank you!